control/timeresp.py

```diff
@@ -332,8 +336,11 @@ def forced_response(sys, T=None, U=0., X0=0., transpose=False,
     # T must be array-like and values must be increasing.
     # The length of T determines the length of the input vector.
     if T is None:
-        raise ValueError('Parameter ``T``: must be array-like, and contain '
-                         '(strictly monotonic) increasing numbers.')
+        if not isdtime(sys, strict=True):
```

Comment:
would `isctime` be simpler and more direct here? 

related: for a system with undefined timebase (dt=None), does it make sense to be lenient and  assume it is a discrete-time system with dt=1 if no T is provided?

Reply:
I just kept it consistent with the block above.

Your proposal is an enhancement and I like it.